classlib/Address.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * Address class
+ *.
+ * @package
+ * @subpackage
+ * @category
+ * @author Sanjeeb Rao
+ */
+
+class Address {
+    private $fields = array(
+        'employee_id' => '',
+        'street' => '',
+        'city' => '',
+        'state' => '',
+        'pin' => '',
+        'phone' => '',
+        'fax' => '',
+        'type' => ''
+    );
+
+    private $db_obj = NULL;
+    private $table_name = 'address';
+
+    /**
+     * Initialization of Database object.
+     *
+     * @access private constructor of Address
+     * @param  object  db_object
+     */
+    public function __construct($db_object)
+    {
+        $this->db_obj = $db_object;
+    }
+
+    /**
+     * Update/create address.
+     *
+     * @access public  update_address
+     * @param  integer emp_id
+     * @param  array   address
+     * @param  string  type
+     * @param  string  time
+     * @return mixed
+     */
+    public function update_address($emp_id, $address, $type, $time = '')
+    {
+       $this->fields['type'] = $type;
+       $this->fields['employee_id'] = $emp_id;
+
+        foreach ($address as $key => $value)
```

Comment:
Use a little more descriptive name which could be associated with the property stored and context